### DISCUSSION TITLE
프로젝트 메니지먼트 (PM) 트레이닝

### DISCUSSION THREAD

**Discussion by bryantson** [Korean-American-IT-Association-KAITA/kaita-project-management#20] [5. 트레이닝/멘토링 이벤트]
기본적인 Agile/Scrum 방식 프로젝트 메니지먼트 트레이닝입니다.

**Comment by j2jun** [comment]
PM 트레이닝 필요합니다.
가능한 시간: 금, 토 8pm-11pm cst

**Comment by klee5760** [comment]
가능한 시간: 오후 9시부터 12시까지 가능합니다.

**Comment by ghost** [comment]
PM 트레이닝 필요합니다.
가능한 시간: 월-목 7pm이후(PST) / 일 6pm이후

**Reply by ghost** [reply]
금/토가 항상 스케줄이 있는것은 아니라서 최대한 시간을 맞춰보겠습니다.

**Comment by Indosaram** [comment]
대부분 미국에 계신분들이 많아서 시간 적기가 애매하네요 ㅠ_ㅠ 일정이 되면 참여하고싶어요

**Comment by yeinsong** [comment]
금, 토 19:00 ~ CST 가능합니다

**Comment by judyyhham** [comment]
월, 화, 금, 토 6pm ~ CST

**Comment by bryantson** [comment]
여기도 왠지 토욜 7PM CST - 10:00 PM CST 쯤으로 맞쳐야 할것 같네요

**Comment by bryantson** [comment]
프로젝트 메니지먼트 트레이닝은 10/09/21 토욜 7PM CST - 8 PM CST그리고 깃허브 트레이닝은 8:30PM CST - 9:30 PM CST 생각합니다

**Reply by ghost** [reply]
브라이언님! 여기 계신분들 모두 금요일 저녁도 시간이 괜찮은거 같은데, 브라이언님만 괜찮으시다면, 혹시 금요일 저녁(kick-off미팅을 피해서)으로 시간조정이 가능할까요? 이 모임에 꼭 참여하고 배우고 싶은데 제가 토요일 저녁늦게까지 일정이 있어요ㅠ

**Reply by bryantson** [reply]
금요일로 한번 맞쳐보겠습니다

**Reply by Ssuny** [reply]
시간이 정해 졌네요. 
최대한 시간 맞춰 참가하도록 하겠습니다.

**Comment by Ssuny** [comment]
목요일, 일요일 9:00 pm 이후 가능합니다.

**Comment by kikibell1004** [comment]
미팅에 참석하고 싶습니다. 
미팅 invitation 보내주실 수 있으신가요? 
가능한 시간에 최대한 맞춰 보도록 하겠습니다.

**Comment by haanmo** [comment]
최대한 미팅 시간에 맞춰서 참석하겠습니다.

**Reply by bryantson** [reply]
감사합니다
